cppcache/src/CacheImpl.hpp

```diff
@@ -134,6 +134,12 @@ class APACHE_GEODE_EXPORT CacheImpl {
    */
   DistributedSystem& getDistributedSystem();
```

Comment:
Any chance this non `const` version can be refactored out?

Reply:
Seems feasible :)